pages/careers/_human-friendly.html

```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
-    how we fund it, how we hire, and how we work.
+    we got to $1 million in annual recurring revenue with $0 in fundraising</a> for all the details on what we do, how we fund it, how we hire, and how we work.</p>
+
+    <h3>Benefits</h3>
+    <p>Our benefits reflect our values. We believe compensation should be fair, transparent, and generous.</p>
+    <ul>
+      <li><strong>Above-Market Salary.</strong> We compute your salary <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">formulaically</a>, using as inputs the average salary for your job title in New York City, the Gruntwork multiplier (currently 1.45) and the cost of living in your home city relative to New York City as reported by numbeo.com. We aim to systematically pay above market.</li>
+      <li><strong>Profit-Sharing Bonus.</strong> We set aside a pot of money at the end of each year based on profits and distribute bonuses according to a formula that uses as inputs your level within the company and the length of your tenure at the company.</li>
+      <li><strong>Performance Bonus.</strong> We give performance-based bonuses of $5,000 - $15,000 as often as once a quarter, depending on your performance.</li>
+      <li><strong>Progressive Equity.</strong> We grant <a href="https://medium.com/detour-dot-com/introducing-progressive-equity-f424a51ee3a4">progressive equity</a> to all new Grunts.</li>
+      <li><strong>Medical Insurance.</strong> We offer high-quality plans, including some with no deductible through JustWorks.com. We cover 80% of your health insurance costs and 55% of those of your family.</li>
+      <li><strong>Dental Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>Vision Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>FSA and HSAs.</strong> We don't contribute to these accounts, but we do offer them as an option.</li>
+      <li><strong>401(k).</strong> We contribute 3% of your salary to your 401(k), regardless of how much you contribute yourself.</li>
+      <li><strong>Disability insurance.</strong> If you get disabled, we have a policy that will pay out a portion of your salary.</li>
+      <li><strong>Hardware budget.</strong> We'll buy you a brand new 16" Apple MacBook Pro upon joining. It will be owned by you, not the company.</li>
+      <li><strong>Personal Budget.</strong> We'll give you a personal budget of $500/month to spend on improving your health, productivity, or knowledge.</li>
+      <li><strong>Minimum Vacation.</strong> We require that you take at least 4 weeks of vacation per year.</li>
+    </ul>
+
+    <p><em>The above benefits reflect our offering package for Grunts based in the USA. For Grunts based in other countries, we make every effort to offer an equivalent total package specific to that country.</em></p>
```

Comment:
This makes all non US employees feel like second class citizens. Remove. Instead, update the above descriptions equivalently.

Reply:
Ah, didn't realize it read that way. Ok, made more changes to most of the above in b47acab.

Reply:
If this is supposed to apply to multiple countries, having exact numbers is even more inaccurate / misleading. If you remove those numbers, then it's more honest and easier to maintain!

Reply:
Ok, you seem to feel pretty strongly about that and agreed we don't want these numbers to get out of date so fixed in 150196cd4140a538d19d84a38b793b0cbfa64d2a.